<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorTestClass</class>
 <widget class="QMainWindow" name="CalculatorTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalculatorTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="button_run">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>131</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Avenir LT 45 Book&quot;;</string>
    </property>
    <property name="text">
     <string>计算</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="text_result">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>20</y>
      <width>511</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Avenir LT 45 Book&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_one">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Avenir LT 45 Book&quot;;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_two">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Avenir LT 45 Book&quot;;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_three">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Avenir LT 45 Book&quot;;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_four">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Avenir LT 45 Book&quot;;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_five">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Avenir LT 45 Book&quot;;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_six">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Avenir LT 45 Book&quot;;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_seven">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Avenir LT 45 Book&quot;;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_eight">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Avenir LT 45 Book&quot;;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_nine">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Avenir LT 45 Book&quot;;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_zero">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>131</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Avenir LT 45 Book&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_plus">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Avenir LT 45 Book&quot;;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_less">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Avenir LT 45 Book&quot;;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="calculatortest.qrc"/>
 </resources>
 <connections/>
</ui>
